deps/v8/tools/gen-postmortem-metadata.py

```diff
@@ -150,7 +146,7 @@
     'Map, instance_size, int, kInstanceSizeOffset',
     'Map, bit_field, char, kBitFieldOffset',
     'Map, bit_field2, char, kBitField2Offset',
-    'Map, bit_field3, int, kBitField3Offset',
+    'Map, bit_field3, SMI, kBitField3Offset',
```

Comment:
4312f8d760bdf574b5a461e426331f0b4d298f18 changed `bit_field3` to be an `int` instead of an `SMI`, so that mdb_v8 can actually inspect its value and determine the correct type depending on which V8 version is used. We'll need to keep this field as an `int` to keep mdb_v8 working.